dspace-api/src/test/java/org/dspace/content/virtual/RelatedTest.java

```diff
@@ -0,0 +1,141 @@
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.UUID;
+
+import org.dspace.content.Entity;
+import org.dspace.content.EntityType;
+import org.dspace.content.Item;
+import org.dspace.content.Relationship;
+import org.dspace.content.RelationshipType;
+import org.dspace.content.service.EntityService;
+import org.dspace.content.service.RelationshipService;
+import org.dspace.core.Context;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RelatedTest {
+
+    @InjectMocks
+    private Related related;
+
+    @Mock
+    private RelationshipService relationshipService;
+
+    @Mock
+    private EntityService entityService;
+
+    @Mock
+    private VirtualMetadataConfiguration virtualMetadataConfiguration;
+
+    @Mock
+    private Context context;
+
+
+    @Test
+    public void testGetRelationshipTypeString() {
+        related.setRelationshipTypeString("TestType");
+        assertEquals("TestGetRelationshipTypeString 0", "TestType", related.getRelationshipTypeString());
+    }
+
+    @Test
+    public void testSetRelationshipTypeString() {
+        related.setRelationshipTypeString("TestType");
+        assertEquals("TestSetRelationshipTypeString 0", "TestType", related.getRelationshipTypeString());
+    }
+
+    @Test
+    public void testSetPlace() {
+        related.setPlace(0);
+        assertTrue("TestSetPlace 0", 0 == related.getPlace());
+    }
+
+    @Test
+    public void testGetPlace() {
+        related.setPlace(0);
+        assertTrue("TestGetPlace 0", 0 == related.getPlace());
+    }
+
+    @Test
+    public void testGetVirtualMetadataConfiguration() {
+        assertEquals("TestGetVirtualMetadataConfiguration 0", virtualMetadataConfiguration.getClass(),
+                related.getVirtualMetadataConfiguration().getClass());
+    }
+
+    @Test
+    public void testSetVirtualMetadataConfiguration() {
+        related.setVirtualMetadataConfiguration(virtualMetadataConfiguration);
+        assertEquals("TestGetVirtualMetadataConfiguration 0", virtualMetadataConfiguration,
+                related.getVirtualMetadataConfiguration());
+    }
+
+    @Test
+    public void testSetUseForPlace() {
+        related.setUseForPlace(true);
+        assertEquals("TestSetVirtualMetadataConfiguration 0", true, related.getUseForPlace());
+    }
+
+    @Test
+    public void testGetUseForPlace() {
+        related.setUseForPlace(true);
+        assertEquals("TestSetVirtualMetadataConfiguration 0", true, related.getUseForPlace());
+    }
+
+    @Test
+    public void testGetValues() throws Exception {
+        List<RelationshipType> relationshipTypeList = new ArrayList<>();
```

Comment:
Again inline comments would be helpful here. There's a lot of content & relationships getting mocked for this test.